src/tests/f2pygen_test.py

```diff
@@ -1077,8 +1077,8 @@ def test_selectiongen_addcase():
     sgen.addcase("1")
     gen = str(sub.root)
     print gen
-    expected = ("SELECT CASE ( my_var )\n"
-                "CASE ( 1 )\n"
+    expected = ("      SELECT CASE ( my_var )\n"
```

Comment:
change needed here as new version of parser adds in spaces appropriately for select.